source/JABHandler.py

```diff
@@ -791,7 +795,7 @@ def event_enterJavaWindow(hwnd):
 def enterJavaWindow_helper(hwnd):
 	vmID=c_long()
 	accContext=JOBJECT64()
-	timeout=time.time()+0.2
+	timeout = time.time() + 0.5

```

Comment:
Could you add a comment why this time is used specifically?

Reply:
This was changed because NVDA would sometimes hang until focus changed.